### ISSUE TITLE
Allow for hide file fieldf (filename) in the board view (dataview based)

### ISSUE THREAD

**Issue by danieltomasz** [marcusolsson/obsidian-projects#830]
# Folder  type project
### What would you like to be added?

 Allowing user to choose which field should be used for card   display - it could be filename but other user chosen field as well

### Why is this needed?

My use case involves managing notes that are generated by a template from an external system. For instance, this could include article annotations imported from Zotero, but the application extends beyond this specific example. Users might generate zettels with alphanumeric codes or have other unique identifiers important to their workflow. The ability to choose a primary display field (other than or in addition to the filename) would greatly enhance the usability and organization of the notes or documents within the app.

I could have alphanumeric title important to other worklfow and use other field for displaying the note (like title): currently user can add additional fields to display, my feature request is about optional replacing filename as main reference in card

# Dataview type project (added later - but might be easier to implement and more useful for me atm)

So basically I want the same abiltity to hide Filetitle the same way as in the table view
At this moment I switched to table view (as only this table is able to read properties from the body of a. file)
I actually use this query
```
TABLE WITHOUT ID   AS "title" 
```
I just want to be able to hide FIle property/filed  in board view, whereas I am able to render `link(file.path, title) AS "title"` as richtext  in the form that suits my needs.

**Comment by danieltomasz** [comment]
So basically I want the same abiltity to hide Filetitle the same way as in the table view
At this moment I switched to table view (as only this table is able to read properties from the body of a. file)
I actually use this query
```
TABLE WITHOUT ID   AS "title" 
```
I just want to be able to hide FIle property/filed  in board view, whereas I am able to render `link(file.path, title) AS "title"` in the form that suits my needs.

**Comment by Acylation** [comment]
Yeah I understand this case since the filename could be pretty complex if its automatically generated. This feature is previously requested in #367 and I'm putting it in the next milestone (my resolution, but can't be 100% sure since the time is really limited).

Based on this I would close this one as duplicated. Let's track the progress in the existing post. Thanks for your engagement and the usecase, I got noticed by mail each time this repo got an update, and I will triaging other issues later.
